app/serializers/submission_status_serializer.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class SubmissionStatusSerializer
+  include JSONAPI::Serializer
+
+  attributes :id, :form_type, :status, :created_at, :updated_at
```

Comment:
Looks like the API can return a `message` and `detail` as well. Are we intentionally omitting those? Seems prudent to omit them, but just wanted to confirm. 🙂

Reply:
I'll add these two fields.

Reply:
Added `message` and `detail`.  Thanks!